x-pack/test/functional/es_archives/rules_scheduled_task_id/rules/data.json

```diff
@@ -114,36 +81,3 @@
     }
   }
 }
-
-{
```

Comment:
Moved to `x-pack/test/functional/es_archives/rules_scheduled_task_id/tasks/data.json`